### DISCUSSION TITLE
How to Format Snippet properly

### DISCUSSION THREAD

**Discussion by usernameyuan** [artisticat1/obsidian-latex-suite#235] [Q&A] [answered]
Hi, thanks for the great plugin, its very well-implemented 

But I am having problems with including my custom plugins for example:
`{
    trigger: "MinkowsiMetric",
    replacement: "\\eta = \\begin{pmatrix}\n-1 & 0 & 0 & 0 \\\n0 & 1 & 0 & 0 \\\n0 & 0 & 1 & 0 \\\n0 & 0 & 0 & 1 \n\\end{pmatrix}",
    options: "m"
}`
says invalid syntax 
Similarly for 
`{trigger: "loretnztensor", replacement: "\\Lambda_{ $0}^{}", options: "mA"}
`

I have looked at the documentation and experimented around but have not been able to deduce the issue 

Thanks for the help!

**Comment by artisticat1** [comment, marked as answer]
Your snippets themselves look fine. Make sure you insert a comma `,` after each snippet though.
